src/core/gdb-server.h

```diff
@@ -10,7 +10,7 @@
  *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
  *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
  *   GNU General Public License for more details.                          *
- *                                                                         *
+ *                                                                         *O
```

Comment:
*cough*

Reply:
It should be fixed, feel free to resolve the conversation. 🐣